src/AciTypeResolver.js

```diff
@@ -304,7 +304,11 @@ class AciTypeResolver {
         }
 
         const def = [
-            ...namespaceData.type_defs,
+            /**
+             * @deprecated ACIs generated by aesophia@7 and below
```

Comment:
It sounds strange because I can't say what to use instead due to `typedef` change is not released yet

Reply:
That was my point in the previous comment, once release we can update this PR and merge it.